.github/workflows/commit_skeleton.yml

```diff
@@ -95,8 +95,9 @@ jobs:
       timeout-minutes: 2
       run: |
         docker exec $rune_test bash -c "while true; do
-        rune state skeleton-enclave-container
-        if [ $? -eq 0 ]; then
+        status=\$(rune list | grep skeleton-enclave-container | awk '{print \$3}')
+        echo Current status: \$status
```

Comment:
I am a little worried about that:
If the system has no skeleton-enclave-container, then `rune list | grep skeleton-enclave-container | awk '{print \$3}'`  will return an error. This step will exit.

I see your success cicd test, we can just see a `running` state.

Please do more tests.